debian/control

```diff
@@ -0,0 +1,18 @@
+Source: ice-setup
+Maintainer: Dan Mick <dan.mick@redhat.com>
+Section: python
+Priority: optional
+Build-Depends: python-setuptools (>= 0.6b3), python-all (>= 2.6.6-3), debhelper (>= 7.4.3)
+Standards-Version: 3.9.1
+X-Python-Version: >=2.6
```

Comment:
Precise has python 2.7 as well, and regardless, I think that's dynamically calculated into .substvars, or should be.  Wheezy might be another question if we revive that.  For right now, I've been told not to worry about precise, but it would be interesting to try this build on precise and see what results; that's easy enough.